docs/operations/hotplug_interfaces.md

```diff
@@ -0,0 +1,98 @@
+# Hotplug Network Interfaces
+KubeVirt now supports hotplugging network interfaces into a running Virtual
+Machine Instance (VMI). The hotplugged interface's model  will be `virtio` and
+it will be of type [bridge](http://kubevirt.io/api-reference/main/definitions.html#_v1_interfacebridge).
+
+When a VM having hotplugged interfaces is rebooted, the hotplugged interfaces
+will be attached to the restarted VM. If these interfaces are persisted, they
```

Comment:
What does it mean for the interfaces to be persisted?
Does the user have to do something in order for that to happen?

Reply:
Check the [persist section](https://github.com/kubevirt/user-guide/pull/652/files#diff-1825ea10641a34a2e6f9d4b2eb70a37462af20304431775e6cc6ae8176082eddR79). 

I could omit it from here, probably that would raise less questions.

Reply:
+1 for removing it from the intro